lib/include/fdoprot.h

```diff
@@ -117,19 +105,13 @@
 #define FDO_TYPE_DEVICE_CRED 6
 #define FDO_TYPE_HMAC 7
 
-// Report
-#define FDO_MGR_AGENT_SND_REPORT 52
-#define FDO_MGR_AGENT_RCV_REPORT_ACK 53
-
 // For restful URL mapping
+// TODO: Not used, needs to review
 #define FDOMsg_typeMIN FDO_TO1_TYPE_HELLO_FDO
 #define FDOMsg_typeMAX FDO_TO2_DONE2
 
-// Protocol version
-#define FDO_VER_MAJOR 1
-#define FDO_VER_MINOR 10
-
 // Error Message
+// TODO: Not used, needs to review (Only INTERNAL_SERVER_ERROR is used)
```

Comment:
should this comment be removed? What is the TODO?

Reply:
Updated the TODO statement that it should be mapped to the error messages.

Reflected in commit 1c24ccaba0e54c28b18fdfaa30a87b1cad086e07